--- thunar-4.17.11/thunar/thunar-settings.desktop.in~	2022-10-28 20:37:00.000000000 +0000
+++ thunar-4.17.11/thunar/thunar-settings.desktop.in	2022-11-13 01:56:14.435826939 +0000
@@ -8,4 +8,5 @@
 Terminal=false
 Type=Application
 Categories=XFCE;GTK;Settings;DesktopSettings;X-XFCE-SettingsDialog;X-XFCE-PersonalSettings;
+OnlyShowIn=XFCE;
 StartupNotify=true
--- thunar-4.17.11/thunar-bulk-rename.desktop.in.in~	2022-10-28 20:37:00.000000000 +0000
+++ thunar-4.17.11/thunar-bulk-rename.desktop.in.in	2022-11-13 01:59:55.452316194 +0000
@@ -8,4 +8,5 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=Utility;Core;GTK;Filesystem;
+Categories=XFCE;System;Utility;Core;GTK;Filesystem;
+OnlyShowIn=XFCE;
--- thunar-4.17.11/thunar.desktop.in.in~	2022-10-28 20:37:00.000000000 +0000
+++ thunar-4.17.11/thunar.desktop.in.in	2022-11-13 02:01:54.252563463 +0000
@@ -8,7 +8,8 @@
 Terminal=false
 StartupNotify=true
 Type=Application
-Categories=System;Core;GTK;FileTools;FileManager;
+Categories=XFCE;System;Utility;Core;GTK;FileTools;FileManager;
+OnlyShowIn=XFCE;
 MimeType=inode/directory;
 Actions=open-home;open-computer;open-trash;
 
