--- ./po/LINGUAS.org	2008-10-27 23:51:48.585954820 +0100
+++ ./po/LINGUAS	2008-10-27 23:52:00.442618940 +0100
@@ -1,2 +1,2 @@
 # set of available languages (in alphabetic order)
-ar be ca cs da de dz el en_GB eo es et eu fi fr gl he hu id it ja ka ko ku lt lv mk nb_NO nl nn_NO pa pl pt_BR pt_PT ro ru sk sq sv tr uk ur zh_CN zh_TW
+ar be ca cs da de dz el en_GB eo es et eu fi fr gl he hu id it ja ka ko ku lt lv mk nb nl nn_NO pa pl pt_BR pt ro ru sk sq sv tr uk ur zh_CN zh_TW
--- ./configure.in.org	2008-10-27 23:50:40.116168343 +0100
+++ ./configure.in	2008-10-27 23:51:32.095954847 +0100
@@ -149,7 +149,7 @@ AC_CHECK_MEMBERS([struct statfs.f_mntonn
 dnl ******************************
 dnl *** Check for i18n support ***
 dnl ******************************
-XDT_I18N([ar be ca cs da de dz el en_GB eo es et eu fi fr gl he hu id it ja ka ko ku lt lv mk nb_NO nl nn_NO pa pl pt_BR pt_PT ro ru sk sq sv tr uk ur zh_CN zh_TW])
+XDT_I18N([ar be ca cs da de dz el en_GB eo es et eu fi fr gl he hu id it ja ka ko ku lt lv mk nb nl nn_NO pa pl pt_BR pt ro ru sk sq sv tr uk ur zh_CN zh_TW])
 
 dnl ***********************************
 dnl *** Check for required packages ***
